src/containers/SearchPage/components/results/SearchContent.jsx

```diff
@@ -82,15 +93,29 @@ const SearchContent = ({ content, locale, t }) => {
               {contentTitle}
             </Link>
           )}
-          {content.supportedLanguages.map(lang => (
-            <SearchContentLanguage
-              key={`${lang}_search_content`}
-              language={lang}
-              content={content}
-              contentType={resourceType.contentType}
-            />
-          ))}
+          <StyledHeaderDiv>
+            {content.id &&
+              userAccess &&
+              userAccess.includes(DRAFT_HTML_SCOPE) && (
```

Comment:
Kan kanskje bruke denne notasjonen.
```suggestion
              userAccess?.includes(DRAFT_HTML_SCOPE) && (
```